src/sentry/static/sentry/app/components/acl/access.tsx

```diff
@@ -104,7 +104,7 @@ class Access extends React.Component<Props> {
     const method = requireAll ? 'every' : 'some';
 
     const hasAccess = !access || access[method](acc => orgAccess.includes(acc));
-    const hasSuperuser = !!config.user.isSuperuser;
+    const hasSuperuser = !!(config.user && config.user.isSuperuser);
```

Comment:
Are we going to upgrade to Typescript 3.7 for optional chaining :P 
cc @markstory

Reply:
Funny enough, we are on typescript 3.7 already. But at least in VSCode it didn't like the chaining operator and we don't use it anywhere else 🤷‍♂